### PR TITLE
t/201: Bulletproofed isDomNode helper when used in iframes. Removed isWindow logic.

### LINKED ISSUE #201: isDomNode() is confusing
Introduced in (without a ticket or at least a commit message): https://github.com/ckeditor/ckeditor5-utils/commit/fafd3d3.

It doesn't check if something is a node. It doesn't do much more than lodash's `isNative()` so I'd like to know why it was needed.

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Bulletproofed `isDomNode` helper when used in iframes. Removed `isWindow` logic from the helper. Closes #201.

### CODE REVIEW

File: src/dom/isdomnode.js [hunk 1]
```diff
@@ -7,14 +7,20 @@
...
  * Checks if the object is a native DOM Node.
  *
  * @param {*} obj
  * @returns {Boolean}
  */
 export default function isDomNode( obj ) {
-	return !!( obj && isNative( obj.addEventListener ) );
+	if ( obj ) {
+		if ( obj.defaultView ) {
+			return obj instanceof obj.defaultView.Document;
```
Comment (Reinmar) [inline src/dom/isdomnode.js]: Is document a node?

Comment (oleq) [reply src/dom/isdomnode.js]: Yes

image

Comment (Reinmar) [reply src/dom/isdomnode.js]: Sorry, I actually wanted to know whether this code is needed at all and whether this code is semantically correct. But now I see that both things are true.

**Comment by Reinmar** [comment]
The existing `isWindow()` made me think about:

image

That may be some direction to consider if the one proposed by you in this PR will turn to be incorrect one day. Something like this should do the job:

```js
isDomNode( obj ) {
    return Object.prototype.toString.apply( obj ).startsWith( '[object HTML' )
}
```

**Pull request merged**